src/parsers/manifest/dash/common/get_clock_offset.ts

```diff
@@ -26,15 +26,23 @@ import log from "../../../../log";
  *
  * For example, a response of 1000 would mean that performance.now() is 1 second
  * behind the server's time.
+ *
+ * Sometimes, it may happen that the server is unreacheable for somes reasons,
+ * if we fall in this case, we will define an offset of 0.
  * @param {string} serverClock
  * @returns {number|undefined}
  */
 export default function getClockOffset(
-  serverClock : string
-) : number|undefined {
+  serverClock: string | null
+): number | undefined {
+  if (serverClock === null) {
+    log.warn("DASH Parser: Error on fetching the clock ressource");
```

Comment:
At this level of the code, we don't really know about that.
Just that no `serverClock` was provided to the function